packages/office-ui-fabric-react/src/components/pickers/examples/Picker.CustomResult.Example.tsx

```diff
@@ -40,11 +35,27 @@ export interface IFullDocumentCardProps {
 
 export interface IDocumentPickerProps extends IBasePickerProps<IFullDocumentCardProps> {}
 
+class DocumentPicker extends BasePickerListBelow<IFullDocumentCardProps, IDocumentPickerProps> {}
+
+const checkboxStyles: Partial<ICheckboxStyles> = { root: { margin: '10px 0' } };
+
+const inputProps = {
+  onFocus: () => console.log('onFocus called'),
+  onBlur: () => console.log('onBlur called'),
+  'aria-label': 'Document Picker',
+};
+
+const pickerSuggestionsProps = {
+  suggestionsHeaderText: 'Suggested Documents',
+  noResultsFoundText: 'No Documents Found',
```

Comment:
While you're here, these should be sentence case.
```suggestion
  'aria-label': 'Document picker',
};

const pickerSuggestionsProps = {
  suggestionsHeaderText: 'Suggested documents',
  noResultsFoundText: 'No documents found',
```